docs/isomorphic-support.md

```diff
@@ -31,6 +31,31 @@ npm pack -- --isomorphic
 npm pack-p -- --isomorphic
 ```
 
+If you are using React18, you need to call `hydrateRoot` instead of `createRoot` to hydrate prerendered HTML.
+Enact CLI provides a global variable `ENACT_PACK_ISOMORPHIC` to selectively call those two APIs in your app's `index.js`.
+If you build with isomorphic option, `ENACT_PACK_ISOMORPHIC` will be `true`, otherwise `false`.
+
+Whithin your **src/index.js** file, add a conditional statement to render or hydrate your app:
+```js
+/* global ENACT_PACK_ISOMORPHIC */
+import {createRoot, hydrateRoot} from 'react-dom/client';
+
+import App from './App';
+
+const appElement = (<App />);
+
+// In a browser environment, render the app to the document.
+if (typeof window !== 'undefined') {
+	if (ENACT_PACK_ISOMORPHIC) {
```

Comment:
a build error occurred,

> Failed to compile.
src/index.js
  Line 9:6:  'ENACT_PACK_ISOMORPHIC' is not defined  no-undef
Search for the keywords to learn more about each error.

Reply:
Please follow the guide.